src/types/GTypes/index.ts

```diff
@@ -85,107 +90,3 @@ export * from "./sprites";
 export * from "./tilesets";
 export * from "./titles";
 export * from "./tokens";
-
-/** base_gold is attached to the G object from `socket.on('start'` in `game.js` */
-export const base_gold = {
```

Comment:
I would say it's out of scope because we are a type project, we shouldn't contain data or values.  
Moreover, I think the the goal is to type the game and not the `data.js` itself.